pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -160,6 +160,20 @@ in {
       };
     } ./setuptools-check-hook.sh) {};
 
+    setuptoolsRustBuildHook = callPackage ({ makePythonHook, setuptools-rust, rust }:
```

Comment:
The hook cannot depend on a rust version. How would we deal with when we need to build against a different rust version?

Reply:
doesn't seem like a issue here, it's just using rust for the functions

Reply:
@Artturin is fully correct here. It really only uses functions there. In the end, only required function is `toRustTargetSpec`.

@FRidh as far as I can see https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/rust/lib/default.nix does not depend on Rust version.